config/mailcoach-sdk.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    /*
+     *  You'll find both the API token and endpoint on Mailcoach'
+     *  API tokens screen in the Mailcoach settings.
+     */
+    'api_token' => env('MAILCOACH_API_TOKEN'),
+
+    'endpoint' => env('MAILCOACH_API_ENDPOINT'),
```

Comment:
You should use this endpoint instead of hardcoded string as above. The mail list code should be separate as we may have multiple - something like this:
```php
[
  'mail_list' => [
    'newsletter' => 'THE_CODE_ABOVE',
  ],
]
```